lexers/embedded/rust.xml

```diff
@@ -171,11 +171,9 @@
         <token type="CommentMultiline"/>
         <push state="comment"/>
       </rule>
-      <rule pattern="r#*&#34;(?:\\.|[^\\;])*&#34;#*">
```

Comment:
```xml
<rule pattern="r#*&#34;(?:\\.|[^\\;])*&#34;#*">
```
Wasn't sure about this, I removed it because
```xml
<rule pattern="(?s)(b?r)(#*)(&#34;.*?&#34;\2)">
```
seems to cover it, unless I'm missing something.